src/index.tsx

```diff
@@ -269,10 +274,17 @@ const GliderComponent = React.forwardRef(
     // Expose the glider instance to the user so they can call the methods too
     React.useImperativeHandle(ref, () => gliderRef.current as GliderMethods);
 
+    const Element = props.containerElement || 'div';
+
     return (
-      <div className="glider-contain">
+      <Element className="glider-contain">
         {props.hasArrows && !props.arrows && (
-          <button role="button" className="glider-prev" id="glider-prev">
+          <button
+            type="button"
+            className="glider-prev"
+            aria-label="Previous"
+            id="glider-prev"
```

Comment:
Can you use @react/auto-id to create the ID? that way if a user uses multiple gliders on a page there won't be duplicate IDs

Reply:
hmm, I actually ran into that problem.
I think adding another lib will cost us some extra kilobytes, how about adding a prop to prefix all ids, i.e. `prefixId="homeSlider"` which will result to `id="homeSlider-glider-prev"` ?

Reply:
the lib is pretty small. I think the kbs are better than another prop

https://bundlephobia.com/result?p=@reach/auto-id@0.12.1